infrastructure/templates/infrastructure/initiative_detail.html

```diff
@@ -106,8 +106,10 @@ <h3>Related Initiatives</h3>
     <div class="row">
         <section class="maximized">
             <h3>Related Projects</h3>
-            <ul class="clean multicolumn">{% for item in object.project_set.all  %}
+            <ul class="clean multicolumn">{% for item in object.project_set.all %}
+               {% if item.published == True %}
```

Comment:
`{% if item.published %}` ought to be enough!

Reply:
Damn, right.